metadata-ingestion/src/datahub/ingestion/source/tableau/tableau.py

```diff
@@ -377,6 +377,11 @@ class TableauConfig(
     )
 
     page_size: int = Field(
+        default=10,
+        description="[advanced] Nu.",
+    )
+
+    number_of_object_in_page: int = Field(
```

Comment:
what's the difference between these? we need better docs for them

Reply:
I need to remove this, not required.